Nancy.Rollbar/RollbarDataFactory.cs

```diff
@@ -78,8 +79,11 @@ public class RollbarDataFactory : IRollbarDataFactory {
                     .ToDictionary(x => x.Key, x => x.Value);
                 if (rr.PostParams.Count == 0) {
                     rr.PostParams = null;
-                }
-                rr.PostBody = context.Request.Body.AsString();
+                }
+                //no "using" statement because I do not want to consume the stream, 
+                //leave it open and let Nancy handle that.
+                rr.PostBody = new StreamReader(context.Request.Body).ReadToEnd();

```

Comment:
You still want to dispose the reader here, so a using statement would be appropriate. You do however, want to leave the underlying stream open. Take a look at https://msdn.microsoft.com/en-us/library/gg712952(v=vs.110).aspx for a constructor that will allow us to do so.